packages/yourloops/test/integration/assert/patient-list.assert.ts

```diff
@@ -530,3 +533,118 @@ export const checkMonitoringAlertsIconsInactiveForFirstPatient = async (): Promi
   const firstRowNoDataIcon = within(dataGridRows).getAllByTestId('no-data-icon')[0]
   expect(firstRowNoDataIcon).toHaveStyle(`color: ${disabledColorAsRgba};`)
 }
+
+export const checkPendingPatientManagementMedicalTeam = async () => {
+  // We go to the pending tab
+  const pendingTab = screen.getByRole('tab', { name: 'Pending' })
+  await userEvent.click(pendingTab)
+
+  // Check that the buttons are disabled
+  expect(screen.getByRole('button', { name: 'Remove patient pending-patient@diabeloop.fr' })).toBeDisabled()
+  expect(screen.getByRole('button', { name: 'Resend invite pending-patient@diabeloop.fr' })).toBeDisabled()
+
+  // We go back to the current tab
+  const currentTab = screen.getByRole('tab', { name: 'Current' })
+  await userEvent.click(currentTab)
+}
+
+export const checkRemovePendingPatientMedicalTeam = async () => {
+  // We go to the pending tab
+  const pendingTab = screen.getByRole('tab', { name: 'Pending' })
+  await userEvent.click(pendingTab)
+
+  // We check that the patient list is correct
+  const pendingPatientList = screen.getByTestId('pending-patient-list-grid')
+  expect(pendingPatientList).toHaveTextContent('Invite sent byDateEmailActions')
+  expect(pendingPatientList).toHaveTextContent('Eric ArdMay 17, 2023pending-patient@diabeloop.frResend inviteCancel')
+  expect(pendingPatientList).toHaveTextContent('Rows per page:101–1 of 1')
+
+  // We open the dialog to cancel a pending invitation
+  const cancelInviteButton = screen.getByRole('button', { name: 'Remove patient pending-patient@diabeloop.fr' })
+  await userEvent.click(cancelInviteButton)
+  const cancelInviteDialog = screen.getByRole('dialog')
+  expect(cancelInviteDialog).toHaveTextContent('Remove Pending Patient from A - MyThirdTeam - to be deleted')
+  expect(cancelInviteDialog).toHaveTextContent('Are you sure you want to remove Pending Patient from A - MyThirdTeam - to be deleted?')
+  expect(cancelInviteDialog).toHaveTextContent('You and the care team will no longer have access to their data.')
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏ The text should be modified in the case of a pending invitation given that the access to the patient's data has not been granted yet

Reply:
I created a ticket for this: https://diabeloop.atlassian.net/browse/YLP-2398 (not complete yet)